spec/controllers/state_file/intake_logins_controller_spec.rb

```diff
@@ -381,7 +382,7 @@
         get :edit, params: params
 
         expect(response.status).to eq(200)
-        expect(response.body).to include "Code verified! Authentication needed to continue."
+        expect(response.body).to include "We found your account! Please confirm your identity to finish signing in."
```

Comment:
can we use the translated strings here? for ex. 

```
I18n.t("translated_string_reference")
```

that way we'll have to update less if the strings change.

Reply:
i think this comment applies to all the tests! i know sometimes we want to test the actual string, so it's okay if it doesn't make sense for all of them.

Reply:
Hey @anisharamnani I did this for all the tests I touched